packages/pn-personafisica-webapp/src/components/Contacts/__test__/CourtesyContacts.test.tsx

```diff
@@ -225,9 +262,23 @@ describe('CourtesyContacts Component', async () => {
       },
     ]);
     // simulate rerendering due to redux changes
+    const updatedDigitalCourtesyAddresses = [
```

Comment:
this is

```
const updatedDigitalCourtesyAddresses = [
 {
   ...defaultPhoneAddress,
   value: +39 + phoneValue
 }
];
```

Reply:
phoneValue already contains +39 so I removed it in this object